City-of-Helsinki/drupal-hdbt
# UHF-5844: Style the phasing paragraph

# [UHF-5844](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5844)


## What was done


* Styles and templates for the new phasing paragraph
* Changed stylelint settings so that scss variables can have '--'

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-5844-phasing-paragraph`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-5844-phasing-paragraph`
* Run `make drush-updb drush-cr`

## How to test



Create a basic page and add pahasing(vaiheistus) paragraph to upper and lower content areas. 

- Test that it works with and without sidebar etc.
- Test also that the numbers are visible when selected
- Test with different amounts of phases 1,2,3+ and make sure that the styles work with each amount
- Test desktop and mobile views

Test also that you're able to add the phasing paragraph to tpr service pages upper and lower content areas, landing pages content area and also inside an accordion item.

## Designers review


* [x] This PR has been visually reviewed by a designer (Sebastian Högnabab)

## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/331
